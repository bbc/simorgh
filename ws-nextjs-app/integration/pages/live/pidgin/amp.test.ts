/**
 * @service pidgin
 * @pathname /pidgin/new_live/c7p765ynk9qt
 */

import runAmpTests from '../ampTests';

describe('AMP', () => {
  describe(pageType, () => {
    runAmpTests();
  });
});
