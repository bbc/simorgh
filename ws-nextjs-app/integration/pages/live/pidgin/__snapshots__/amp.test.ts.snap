// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP New_live Header I can see the branding 1`] = `".css-1o8f8s9-BrandSvg{box-sizing:content-box;color:#FFFFFF;fill:currentColor;height:1.25rem;}@media (min-width: 25rem){.css-1o8f8s9-BrandSvg{height:1.5rem;}}@media (min-width: 37.5rem){.css-1o8f8s9-BrandSvg{height:1.875rem;}}@media screen and (-ms-high-contrast: active),print{.css-1o8f8s9-BrandSvg{fill:windowText;}}/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi9Vc2Vycy9haGNob3MwMS9EZXNrdG9wL1dPUkxEIE5FV1Mvc2ltb3JnaC9zcmMvYXBwL2xlZ2FjeS9wc2FtbWVhZC9wc2FtbWVhZC1icmFuZC9zcmMvaW5kZXguanN4Il0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQXNHMkIiLCJmaWxlIjoiL1VzZXJzL2FoY2hvczAxL0Rlc2t0b3AvV09STEQgTkVXUy9zaW1vcmdoL3NyYy9hcHAvbGVnYWN5L3BzYW1tZWFkL3BzYW1tZWFkLWJyYW5kL3NyYy9pbmRleC5qc3giLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgUmVhY3QsIHsgZm9yd2FyZFJlZiB9IGZyb20gJ3JlYWN0JztcbmltcG9ydCBzdHlsZWQgZnJvbSAnQGVtb3Rpb24vc3R5bGVkJztcbmltcG9ydCB7IHN0cmluZywgbnVtYmVyLCBub2RlLCBzaGFwZSwgYm9vbCB9IGZyb20gJ3Byb3AtdHlwZXMnO1xuaW1wb3J0IHtcbiAgR0VMX0dST1VQXzBfU0NSRUVOX1dJRFRIX01BWCxcbiAgR0VMX0dST1VQXzJfU0NSRUVOX1dJRFRIX01JTixcbiAgR0VMX0dST1VQXzNfU0NSRUVOX1dJRFRIX01JTixcbn0gZnJvbSAnI3BzYW1tZWFkL2dlbC1mb3VuZGF0aW9ucy9zcmMvYnJlYWtwb2ludHMnO1xuaW1wb3J0IHtcbiAgR0VMX1NQQUNJTkdfSExGLFxuICBHRUxfU1BBQ0lORyxcbiAgR0VMX1NQQUNJTkdfREJMLFxufSBmcm9tICcjcHNhbW1lYWQvZ2VsLWZvdW5kYXRpb25zL3NyYy9zcGFjaW5ncyc7XG5pbXBvcnQgeyBmb2N1c0luZGljYXRvclRoaWNrbmVzcyB9IGZyb20gJy4uLy4uLy4uLy4uL2NvbXBvbmVudHMvVGhlbWVQcm92aWRlci9mb2N1c0luZGljYXRvcic7XG5pbXBvcnQgVmlzdWFsbHlIaWRkZW5UZXh0IGZyb20gJy4uLy4uLy4uLy4uL2NvbXBvbmVudHMvVmlzdWFsbHlIaWRkZW5UZXh0JztcblxuY29uc3QgU1ZHX1dSQVBQRVJfTUFYX1dJRFRIX0FCT1ZFXzEyODBQWCA9ICc2M3JlbSc7XG5jb25zdCBTQ1JJUFRfTElOS19PRkZTRVRfQkVMT1dfMjQwUFggPSA1MjtcbmNvbnN0IFBBRERJTkdfQVJPVU5EX1NWR19CRUxPV180MDBQWCA9IDE2O1xuXG5jb25zdCBUUkFOU1BBUkVOVF9CT1JERVIgPSBgMC4wNjI1cmVtIHNvbGlkIHRyYW5zcGFyZW50YDtcblxuY29uc3QgU3ZnV3JhcHBlciA9IHN0eWxlZC5kaXZgXG4gIGhlaWdodDogMTAwJTtcbiAgcG9zaXRpb246IHJlbGF0aXZlO1xuICBkaXNwbGF5OiBmbGV4O1xuICBqdXN0aWZ5LWNvbnRlbnQ6IHNwYWNlLWJldHdlZW47XG4gIGFsaWduLWl0ZW1zOiBjZW50ZXI7XG4gIGZsZXgtd3JhcDogd3JhcDtcbiAgbWF4LXdpZHRoOiAke1NWR19XUkFQUEVSX01BWF9XSURUSF9BQk9WRV8xMjgwUFh9O1xuICBtYXJnaW46IDAgYXV0bztcblxuICBAbWVkaWEgKG1heC13aWR0aDogJHtHRUxfR1JPVVBfMF9TQ1JFRU5fV0lEVEhfTUFYfSkge1xuICAgIGRpc3BsYXk6IGJsb2NrO1xuICB9XG5gO1xuXG5jb25zdCBCYW5uZXIgPSBzdHlsZWQuZGl2YFxuICBiYWNrZ3JvdW5kLWNvbG9yOiAke3Byb3BzID0+IHByb3BzLnRoZW1lLnBhbGV0dGUuQlJBTkRfQkFDS0dST1VORH07XG4gIGhlaWdodDogJHs0NCAvIDE2fXJlbTtcbiAgd2lkdGg6IDEwMCU7XG4gIHBhZGRpbmc6IDAgJHtHRUxfU1BBQ0lOR307XG5cbiAgQG1lZGlhIChtaW4td2lkdGg6ICR7R0VMX0dST1VQXzJfU0NSRUVOX1dJRFRIX01JTn0pIHtcbiAgICBoZWlnaHQ6ICR7NTYgLyAxNn1yZW07XG4gICAgcGFkZGluZzogMCAke0dFTF9TUEFDSU5HX0RCTH07XG4gIH1cblxuICBAbWVkaWEgKG1pbi13aWR0aDogJHtHRUxfR1JPVVBfM19TQ1JFRU5fV0lEVEhfTUlOfSkge1xuICAgIGhlaWdodDogJHs2NCAvIDE2fXJlbTtcbiAgfVxuXG4gIEBtZWRpYSAobWF4LXdpZHRoOiAke0dFTF9HUk9VUF8wX1NDUkVFTl9XSURUSF9NQVh9KSB7XG4gICAgJHsoeyBzY3JpcHRMaW5rLCBzdmdIZWlnaHQgfSkgPT5cbiAgICAgIHNjcmlwdExpbmsgJiZcbiAgICAgIGBtaW4taGVpZ2h0OiAke1xuICAgICAgICAoc3ZnSGVpZ2h0ICtcbiAgICAgICAgICBQQURESU5HX0FST1VORF9TVkdfQkVMT1dfNDAwUFggK1xuICAgICAgICAgIFNDUklQVF9MSU5LX09GRlNFVF9CRUxPV18yNDBQWCkgL1xuICAgICAgICAxNlxuICAgICAgfXJlbTtgfVxuICB9XG5cbiAgJHsoeyBib3JkZXJUb3AgfSkgPT4gYm9yZGVyVG9wICYmIGBib3JkZXItdG9wOiAke1RSQU5TUEFSRU5UX0JPUkRFUn1gfTtcbiAgJHsoeyBib3JkZXJCb3R0b20gfSkgPT5cbiAgICBib3JkZXJCb3R0b20gJiYgYGJvcmRlci1ib3R0b206ICR7VFJBTlNQQVJFTlRfQk9SREVSfWB9O1xuYDtcblxuY29uc3Qgc3R5bGVkTGlua091dGxpbmUgPSBgXG4gIGNvbnRlbnQ6ICcnO1xuICBwb3NpdGlvbjogYWJzb2x1dGU7XG4gIHRvcDogMDtcbiAgbGVmdDogLSR7Zm9jdXNJbmRpY2F0b3JUaGlja25lc3N9O1xuICBib3R0b206IDA7XG4gIHJpZ2h0OiAtJHtmb2N1c0luZGljYXRvclRoaWNrbmVzc307XG4gIGA7XG5cbmNvbnN0IFN0eWxlZExpbmsgPSBzdHlsZWQuYWBcbiAgaGVpZ2h0OiAxMDAlO1xuICBkaXNwbGF5OiBmbGV4O1xuICBhbGlnbi1pdGVtczogY2VudGVyO1xuICBwb3NpdGlvbjogcmVsYXRpdmU7XG4gIGJvdHRvbTogMC4xMjVyZW07XG4gIHBhZGRpbmctdG9wOiAwLjEyNXJlbTtcbiAgJjpob3ZlcixcbiAgJjpmb2N1cyB7XG4gICAgdGV4dC1kZWNvcmF0aW9uOiBub25lO1xuICAgIGJvcmRlci1ib3R0b206ICR7R0VMX1NQQUNJTkdfSExGfSBzb2xpZFxuICAgICAgJHtwcm9wcyA9PiBwcm9wcy50aGVtZS5wYWxldHRlLkJSQU5EX0xPR099O1xuICAgIG1hcmdpbi1ib3R0b206IC0ke0dFTF9TUEFDSU5HX0hMRn07XG4gIH1cblxuICAvKiBDdXN0b20gZm9jdXMgaW5kaWNhdG9yIHN0eWxpbmcgYXBwbGllZCB0byBwc2V1ZG8tZWxlbWVudC4gR2xvYmFsIGZvY3VzIGluZGljYXRvciBzdHlsaW5nIGhhcyBiZWVuIHJlbW92ZWQuICovXG4gICY6Zm9jdXMtdmlzaWJsZTo6YWZ0ZXIge1xuICAgICR7c3R5bGVkTGlua091dGxpbmV9XG4gICAgYm9yZGVyLXRvcDogJHtHRUxfU1BBQ0lOR19ITEZ9IHNvbGlkICR7cHJvcHMgPT5cbiAgICAgIHByb3BzLnRoZW1lLnBhbGV0dGUuQlJBTkRfTE9HT307XG4gICAgb3V0bGluZTogJHtHRUxfU1BBQ0lOR19ITEZ9IHNvbGlkICR7cHJvcHMgPT4gcHJvcHMudGhlbWUucGFsZXR0ZS5CUkFORF9MT0dPfTtcbiAgfVxuYDtcblxuLy8gYGN1cnJlbnRDb2xvcmAgaGFzIGJlZW4gdXNlZCB0byBhZGRyZXNzIGhpZ2ggY29udHJhc3QgbW9kZSBpbiBGaXJlZm94LlxuY29uc3QgQnJhbmRTdmcgPSBzdHlsZWQuc3ZnYFxuICBib3gtc2l6aW5nOiBjb250ZW50LWJveDtcbiAgY29sb3I6ICR7cHJvcHMgPT4gcHJvcHMudGhlbWUucGFsZXR0ZS5CUkFORF9MT0dPfTtcbiAgZmlsbDogY3VycmVudENvbG9yO1xuICBoZWlnaHQ6ICR7MjAgLyAxNn1yZW07XG5cbiAgQG1lZGlhIChtaW4td2lkdGg6ICR7R0VMX0dST1VQXzJfU0NSRUVOX1dJRFRIX01JTn0pIHtcbiAgICBoZWlnaHQ6ICR7MjQgLyAxNn1yZW07XG4gIH1cblxuICBAbWVkaWEgKG1pbi13aWR0aDogJHtHRUxfR1JPVVBfM19TQ1JFRU5fV0lEVEhfTUlOfSkge1xuICAgIGhlaWdodDogJHszMCAvIDE2fXJlbTtcbiAgfVxuXG4gIEBtZWRpYSBzY3JlZW4gYW5kICgtbXMtaGlnaC1jb250cmFzdDogYWN0aXZlKSwgcHJpbnQge1xuICAgIGZpbGw6IHdpbmRvd1RleHQ7XG4gIH1cbmA7XG5cbmNvbnN0IExvY2FsaXNlZEJyYW5kTmFtZSA9ICh7IGxpbmtJZCwgcHJvZHVjdCwgc2VydmljZUxvY2FsaXNlZE5hbWUgfSkgPT4ge1xuICBjb25zdCBicmFuZElkID0gYEJyYW5kTGluay0ke2xpbmtJZH1gO1xuICByZXR1cm4gc2VydmljZUxvY2FsaXNlZE5hbWUgPyAoXG4gICAgLy8gaWQ9e2BCcmFuZExpbmstJHtsaW5rSWR9YCBpcyBhIHRlbXBvcmFyeSBmaXggZm9yIHRoZSBhMTF5IG5lc3RlZCBzcGFuJ3MgYnVnIGV4cGVyaWVuY2VkIGluIFRhbGtCYWNrLCByZWZlciB0byB0aGUgZm9sbG93aW5nIGlzc3VlOiBodHRwczovL2dpdGh1Yi5jb20vYmJjL3NpbW9yZ2gvaXNzdWVzLzk2NTJcbiAgICAvLyBlc2xpbnQtZGlzYWJsZS1uZXh0LWxpbmUganN4LWExMXkvYXJpYS1yb2xlXG4gICAgPFZpc3VhbGx5SGlkZGVuVGV4dCByb2xlPVwidGV4dFwiIGlkPXticmFuZElkfT5cbiAgICAgIDxzcGFuIGxhbmc9XCJlbi1HQlwiPntgJHtwcm9kdWN0fSwgYH08L3NwYW4+XG4gICAgICA8c3Bhbj57c2VydmljZUxvY2FsaXNlZE5hbWV9PC9zcGFuPlxuICAgIDwvVmlzdWFsbHlIaWRkZW5UZXh0PlxuICApIDogKFxuICAgIDxWaXN1YWxseUhpZGRlblRleHQgaWQ9e2JyYW5kSWR9Pntwcm9kdWN0fTwvVmlzdWFsbHlIaWRkZW5UZXh0PlxuICApO1xufTtcblxuTG9jYWxpc2VkQnJhbmROYW1lLnByb3BUeXBlcyA9IHtcbiAgbGlua0lkOiBzdHJpbmcuaXNSZXF1aXJlZCxcbiAgcHJvZHVjdDogc3RyaW5nLmlzUmVxdWlyZWQsXG4gIHNlcnZpY2VMb2NhbGlzZWROYW1lOiBzdHJpbmcsXG59O1xuXG5Mb2NhbGlzZWRCcmFuZE5hbWUuZGVmYXVsdFByb3BzID0ge1xuICBzZXJ2aWNlTG9jYWxpc2VkTmFtZTogbnVsbCxcbn07XG5cbmNvbnN0IFN0eWxlZEJyYW5kID0gKHsgbGlua0lkLCBwcm9kdWN0LCBzZXJ2aWNlTG9jYWxpc2VkTmFtZSwgc3ZnIH0pID0+IChcbiAgPD5cbiAgICB7c3ZnICYmIChcbiAgICAgIDw+XG4gICAgICAgIDxCcmFuZFN2Z1xuICAgICAgICAgIHZpZXdCb3g9e1tcbiAgICAgICAgICAgIHN2Zy52aWV3Ym94Lm1pblggfHwgMCxcbiAgICAgICAgICAgIHN2Zy52aWV3Ym94Lm1pblkgfHwgMCxcbiAgICAgICAgICAgIHN2Zy52aWV3Ym94LndpZHRoLFxuICAgICAgICAgICAgc3ZnLnZpZXdib3guaGVpZ2h0LFxuICAgICAgICAgIF0uam9pbignICcpfVxuICAgICAgICAgIHhtbG5zPVwiaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmdcIlxuICAgICAgICAgIGZvY3VzYWJsZT1cImZhbHNlXCJcbiAgICAgICAgICBhcmlhLWhpZGRlbj1cInRydWVcIlxuICAgICAgICAgIGhlaWdodD1cIjMyXCJcbiAgICAgICAgPlxuICAgICAgICAgIHtzdmcuZ3JvdXB9XG4gICAgICAgIDwvQnJhbmRTdmc+XG4gICAgICAgIDxMb2NhbGlzZWRCcmFuZE5hbWVcbiAgICAgICAgICBsaW5rSWQ9e2xpbmtJZH1cbiAgICAgICAgICBwcm9kdWN0PXtwcm9kdWN0fVxuICAgICAgICAgIHNlcnZpY2VMb2NhbGlzZWROYW1lPXtzZXJ2aWNlTG9jYWxpc2VkTmFtZX1cbiAgICAgICAgLz5cbiAgICAgIDwvPlxuICAgICl9XG4gIDwvPlxuKTtcblxuY29uc3QgYnJhbmRQcm9wcyA9IHtcbiAgbGlua0lkOiBzdHJpbmcuaXNSZXF1aXJlZCxcbiAgcHJvZHVjdDogc3RyaW5nLmlzUmVxdWlyZWQsXG4gIHNlcnZpY2VMb2NhbGlzZWROYW1lOiBzdHJpbmcsXG4gIHN2Zzogc2hhcGUoe1xuICAgIGdyb3VwOiBub2RlLmlzUmVxdWlyZWQsXG4gICAgcmF0aW86IG51bWJlci5pc1JlcXVpcmVkLFxuICAgIHZpZXdib3g6IHNoYXBlKHtcbiAgICAgIGhlaWdodDogbnVtYmVyLmlzUmVxdWlyZWQsXG4gICAgICB3aWR0aDogbnVtYmVyLmlzUmVxdWlyZWQsXG4gICAgfSkuaXNSZXF1aXJlZCxcbiAgfSkuaXNSZXF1aXJlZCxcbn07XG5cblN0eWxlZEJyYW5kLnByb3BUeXBlcyA9IGJyYW5kUHJvcHM7XG5cblN0eWxlZEJyYW5kLmRlZmF1bHRQcm9wcyA9IHtcbiAgc2VydmljZUxvY2FsaXNlZE5hbWU6IG51bGwsXG59O1xuXG5jb25zdCBCcmFuZCA9IGZvcndhcmRSZWYoKHByb3BzLCByZWYpID0+IHtcbiAgY29uc3Qge1xuICAgIHN2Z0hlaWdodCxcbiAgICBtYXhXaWR0aCxcbiAgICBtaW5XaWR0aCxcbiAgICB1cmwsXG4gICAgYm9yZGVyVG9wLFxuICAgIGJvcmRlckJvdHRvbSxcbiAgICBzY3JpcHRMaW5rLFxuICAgIHNraXBMaW5rLFxuICAgIGxpbmtJZCxcbiAgICAuLi5yZXN0XG4gIH0gPSBwcm9wcztcblxuICByZXR1cm4gKFxuICAgIDxCYW5uZXJcbiAgICAgIHN2Z0hlaWdodD17c3ZnSGVpZ2h0fVxuICAgICAgYm9yZGVyVG9wPXtib3JkZXJUb3B9XG4gICAgICBib3JkZXJCb3R0b209e2JvcmRlckJvdHRvbX1cbiAgICAgIHNjcmlwdExpbms9e3NjcmlwdExpbmt9XG4gICAgICB7Li4ucmVzdH1cbiAgICA+XG4gICAgICA8U3ZnV3JhcHBlciByZWY9e3JlZn0+XG4gICAgICAgIHt1cmwgPyAoXG4gICAgICAgICAgPFN0eWxlZExpbmtcbiAgICAgICAgICAgIGhyZWY9e3VybH1cbiAgICAgICAgICAgIGlkPXtsaW5rSWR9XG4gICAgICAgICAgICBjbGFzc05hbWU9XCJmb2N1c0luZGljYXRvclJlbW92ZVwiXG4gICAgICAgICAgICAvLyBUaGlzIGlzIGEgdGVtcG9yYXJ5IGZpeCBmb3IgdGhlIGExMXkgbmVzdGVkIHNwYW4ncyBidWcgZXhwZXJpZW5jZWQgaW4gVGFsa0JhY2ssIHJlZmVyIHRvIHRoZSBmb2xsb3dpbmcgaXNzdWU6IGh0dHBzOi8vZ2l0aHViLmNvbS9iYmMvc2ltb3JnaC9pc3N1ZXMvOTY1MlxuICAgICAgICAgICAgYXJpYS1sYWJlbGxlZGJ5PXtgQnJhbmRMaW5rLSR7bGlua0lkfWB9XG4gICAgICAgICAgPlxuICAgICAgICAgICAgPFN0eWxlZEJyYW5kIHsuLi5wcm9wc30gLz5cbiAgICAgICAgICA8L1N0eWxlZExpbms+XG4gICAgICAgICkgOiAoXG4gICAgICAgICAgPFN0eWxlZEJyYW5kIHsuLi5wcm9wc30gLz5cbiAgICAgICAgKX1cbiAgICAgICAge3NraXBMaW5rfVxuICAgICAgICB7c2NyaXB0TGluayAmJiA8ZGl2PntzY3JpcHRMaW5rfTwvZGl2Pn1cbiAgICAgIDwvU3ZnV3JhcHBlcj5cbiAgICA8L0Jhbm5lcj5cbiAgKTtcbn0pO1xuXG5CcmFuZC5kZWZhdWx0UHJvcHMgPSB7XG4gIHVybDogbnVsbCxcbiAgc2VydmljZUxvY2FsaXNlZE5hbWU6IG51bGwsXG4gIGJvcmRlclRvcDogZmFsc2UsXG4gIGJvcmRlckJvdHRvbTogZmFsc2UsXG4gIHNjcmlwdExpbms6IG51bGwsXG4gIHNraXBMaW5rOiBudWxsLFxuICBsaW5rSWQ6IG51bGwsXG59O1xuXG5CcmFuZC5wcm9wVHlwZXMgPSB7XG4gIC4uLmJyYW5kUHJvcHMsXG4gIHVybDogc3RyaW5nLFxuICBzZXJ2aWNlTG9jYWxpc2VkTmFtZTogc3RyaW5nLFxuICBib3JkZXJUb3A6IGJvb2wsXG4gIGJvcmRlckJvdHRvbTogYm9vbCxcbiAgc2NyaXB0TGluazogbm9kZSxcbiAgc2tpcExpbms6IG5vZGUsXG4gIGxpbmtJZDogc3RyaW5nLFxufTtcblxuZXhwb3J0IGRlZmF1bHQgQnJhbmQ7XG4iXX0= */.css-49uz87-visuallyHiddenText{-webkit-clip-path:inset(100%);clip-path:inset(100%);clip:rect(1px, 1px, 1px, 1px);height:1px;overflow:hidden;position:absolute;width:1px;margin:0;}/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi9Vc2Vycy9haGNob3MwMS9EZXNrdG9wL1dPUkxEIE5FV1Mvc2ltb3JnaC9zcmMvYXBwL2NvbXBvbmVudHMvVmlzdWFsbHlIaWRkZW5UZXh0L2luZGV4LnN0eWxlcy50c3giXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBSUkiLCJmaWxlIjoiL1VzZXJzL2FoY2hvczAxL0Rlc2t0b3AvV09STEQgTkVXUy9zaW1vcmdoL3NyYy9hcHAvY29tcG9uZW50cy9WaXN1YWxseUhpZGRlblRleHQvaW5kZXguc3R5bGVzLnRzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCB7IGNzcyB9IGZyb20gJ0BlbW90aW9uL3JlYWN0JztcblxuY29uc3Qgc3R5bGVzID0ge1xuICB2aXN1YWxseUhpZGRlblRleHQ6ICgpID0+XG4gICAgY3NzKHtcbiAgICAgIGNsaXBQYXRoOiAnaW5zZXQoMTAwJSknLFxuICAgICAgY2xpcDogJ3JlY3QoMXB4LCAxcHgsIDFweCwgMXB4KScsXG4gICAgICBoZWlnaHQ6ICcxcHgnLFxuICAgICAgb3ZlcmZsb3c6ICdoaWRkZW4nLFxuICAgICAgcG9zaXRpb246ICdhYnNvbHV0ZScsXG4gICAgICB3aWR0aDogJzFweCcsXG4gICAgICBtYXJnaW46ICcwJyxcbiAgICB9KSxcbn07XG5cbmV4cG9ydCBkZWZhdWx0IHN0eWxlcztcbiJdfQ== */BBC News, Pidgin"`;

exports[`AMP New_live Header Navigation link should match text and url 1`] = `
{
  "text": "Home",
  "url": "/pidgin",
}
`;

exports[`AMP New_live Header Navigation link should match text and url 2`] = `
{
  "text": "Nigeria",
  "url": "/pidgin/topics/c2dwqd1zr92t",
}
`;

exports[`AMP New_live Header Navigation link should match text and url 3`] = `
{
  "text": "Africa",
  "url": "/pidgin/topics/c404v061z85t",
}
`;

exports[`AMP New_live Header Navigation link should match text and url 4`] = `
{
  "text": "World",
  "url": "/pidgin/topics/c0823e52dd0t",
}
`;

exports[`AMP New_live Header Navigation link should match text and url 5`] = `
{
  "text": "Video",
  "url": "/pidgin/topics/c3l19z3k1ert",
}
`;

exports[`AMP New_live Header Navigation link should match text and url 6`] = `
{
  "text": "Sport",
  "url": "/pidgin/topics/cjgn7gv77vrt",
}
`;

exports[`AMP New_live Header Navigation link should match text and url 7`] = `
{
  "text": "Entertainment",
  "url": "/pidgin/topics/cqywjyzk2vyt",
}
`;

exports[`AMP New_live Header Navigation link should match text and url 8`] = `
{
  "text": "Most popular",
  "url": "/pidgin/popular/read",
}
`;
